src/js/components/chat-input.jsx

```diff
@@ -17,11 +17,16 @@ export class ChatInputComponent extends Component {
     }
 
     onChange(e) {
+        checkAndResetUnreadCount();
```

Comment:
true, it needs to be throttled, unless we optimisticly put to to 0 before the call and when it comes back it will get updated with the real value?